src/os/interaction/modifyinteraction.js

```diff
@@ -225,6 +223,34 @@ class Modify extends OLModify {
     this.removeControls();
   }
 
+  /**
+   * @inheritDoc
+   */
+  setActive(value) {
+    const changed = value !== this.getActive();
+
+    if (changed && this.clone_) {
+      if (!value) {
+        const geometry = this.clone_.getGeometry();
+        if (geometry) {
+          olEvents.unlisten(geometry, OLEventType.CHANGE, this.onGeometryChange, this);
+        }
+
+        os.MapContainer.getInstance().removeFeature(this.clone_);
```

Comment:
There are a couple global references to `os.MapContainer` here that should use the module import.

Reply:
Unfortunately this file already had some non-modularized references that were necessary due to a non-obvious circular dependency. When I first put it in, the idea was to just use the drawing layer/source directly, but for some reason that didn't play nicely with some of the drawing layer specific-stuff in `MapContainer`.

Reply:
Ahh, ok we can resolve that separately.